openpower/package/petitboot/petitboot.mk

```diff
@@ -19,6 +19,8 @@ PETITBOOT_CONF_OPTS += --with-ncurses --without-twin-x11 --without-twin-fbdev \
 	      HOST_PROG_SHUTDOWN=/usr/libexec/petitboot/bb-kexec-reboot \
 	      $(if $(BR2_PACKAGE_BUSYBOX),--with-tftp=busybox)
 
+PETITBOOT_AUTORECONF_ENV += PETITBOOT_VERSION=`cat $(PETITBOOT_VERSION_FILE) | cut -d '-' -f 2-`
```

Comment:
Skiboot (and it looks like Hostboot too?) sets an environment variable (eg. SKIBOOT_VERSION) to be recognised at build time. Petitboot however determines it's version string during the autoreconf step before configuring (ie. in AC_INIT), so we need to set this variable such that it is recognised *then* rather than at build time.